Sources/Commands/SwiftRunCommand.swift

```diff
@@ -242,11 +244,23 @@ public struct SwiftRunCommand: AsyncSwiftCommand {
                     arguments = options.arguments
                 }
 
+                // For Linux, need to point LD_LIBRARY_PATH at the swift runtime
+                let environment: [String: String]?
+                if toolchain.targetTriple.isLinux() {
```

Comment:
nit: could be `if toolchain.targetTriple.objectFormat == .elf` to cover other relevant platforms as well

(can also be a follow-on change)